app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
+  <div id="error_explanation" data-turbo-cache="false">
+    <h2>
+      <%= I18n.t("errors.messages.not_saved",
+                 count: resource.errors.count,
+                 resource: resource.class.model_name.human.downcase)
+       %>
+    </h2>
+    <ul>
+      <% resource.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  </div>
```

Comment:
- [Required] Good job adding `devise` gem to your project, however, I think it would be better if you also styled the `registration`, `login` & `password reset` forms as illustrated in the project wireframe. As a developer you'll be required to follow design guidelines provided by clients in the future, please see the screenshot below for more details on this, especially the `input` fields and the  `button`.

## Project requirements
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)

## Your design
![Screenshot from 2023-04-20 20-35-04](https://user-images.githubusercontent.com/16392046/233444720-7b6ebfa4-c2cd-4dc6-b402-093c94e2628f.png)

![Screenshot from 2023-04-20 20-38-34](https://user-images.githubusercontent.com/16392046/233445185-5fb608b3-c1a0-4d3a-a69f-5836cda5d712.png)